astroquery/cosmosim/core.py

```diff
@@ -641,7 +630,8 @@ def _starttime_dict(self):
                  for i in range(len(response_list))]
         self.starttime_dict = {}
         for i in range(len(soups)):
-            self.starttime_dict['{}'.format(completed_ids[i])] = '{}'.format(soups[i].find('uws:starttime').string)
+            self.starttime_dict['{}'.format(completed_ids[i])] = (
+                '{}'.format(soups[i].find('uws:starttime').string))
```

Comment:
these could also be simplified to `str(completed_ids[i])` and `str(soups[i].find('uws:starttime').string)`, I suppose?  They're acceptable as is though.